packages/atomic/cypress/e2e/smart-snippet.cypress.ts

```diff
@@ -254,33 +251,28 @@ describe('Smart Snippet Test Suites', () => {
         .init();
     });
 
-    SmartSnippetAssertions.assertShowMore(true);
-    SmartSnippetAssertions.assertShowLess(false);
-    SmartSnippetAssertions.assertCollapseWrapperHeight(heightWhenCollapsed);
-    CommonAssertions.assertAccessibility(smartSnippetComponent);
+    it('should render correctly', () => {
+      SmartSnippetAssertions.assertShowMore(true);
+      SmartSnippetAssertions.assertShowLess(false);
+      SmartSnippetAssertions.assertCollapseWrapperHeight(heightWhenCollapsed);
+    });
 
-    describe('then pressing show more', () => {
-      beforeEach(() => {
-        SmartSnippetSelectors.showMoreButton().click();
-      });
+    CommonAssertions.assertAccessibility(smartSnippetComponent);
 
+    it('should work correctly when pressing show more and show less', () => {
+      SmartSnippetSelectors.showMoreButton().click();
+      cy.wait(200);
```

Comment:
We should avoid using `cy.wait` unless we're asserting that something doesn't happen IMO. `cy.wait` can cause flakiness and slows down tests.

Instead, we should leverage Cypress' retry-ability mechanism ([docs](https://docs.cypress.io/guides/core-concepts/retry-ability)). In other words, the next assertion should be automatically repeated until it passes up to a certain timeout. This may already be the case.

Reply:
The reason, I put cy.wait here is that after the showMoreButton().click() there is an animation and the assert directly after asserts a precise pixel amount (`SmartSnippetAssertions.assertAnswerHeight(height);`)

Reply:
You should be able to wait for the exact pixel amount. If not we should relax or remove the test.

Reply:
The problem here seems to be that `SmartSnippetAssertions.assertAnswerHeight(height);` isn't leveraging Cypress' retry-ability mechanism.

When I originally built this assertion, I recall making sure that it works with the retry-ability mechanism, meaning that, if the height didn't match, it would wait and try again up until Cypress' timeout.

However, it looks like something must've broken it somehow...